app/Exceptions/Handler.php

```diff
@@ -47,6 +49,10 @@ public function report(Exception $exception)
 	 */
 	public function render($request, Exception $exception)
 	{
+		if ($exception instanceof DecryptException && $exception->getMessage() === 'The payload is invalid.') {
+			return response()->json(['error' => 'Session timed out'], 400);
```

Comment:
I'm starting to think that we should consider uniforming the output format of Lychee and Lychee-Laravel to a Json array in all cases instead of just returning an id or true or false or an Json array...
But this should obviously not be done in that PR. :)